common/Makefile

```diff
@@ -22,7 +22,7 @@ clean:
 
 oshinko-cli:
 	- mkdir -p $(clonepath)
-	- cd $(clonepath) && git clone https://github.com/radanalyticsio/oshinko-cli
+	- cd $(clonepath) && git clone -b v0.1.0 https://github.com/radanalyticsio/oshinko-cli
```

Comment:
It works for tags, and puts you in a detached head state.  Good enough for a build.